### DISCUSSION TITLE
S3 Storage For Multiple Instances

### DISCUSSION THREAD

**Discussion by Infinil** [pocketbase/pocketbase#5748] [Q&A]
I wanted to ask if it's okay to connect multiple pocketbase instances (same version) to one S3 storage. Since, I believe it is not possible to specify subfolder/directory.

**Comment by ganigeorgiev** [comment]
It is fine as long as you use a different bucket for each instance.

The only S3 service that I've tested in the past that doesn't support multiple buckets is Digital Ocean Spaces.
Vultr, Wasabi, Backblaze, AWS, MinIO, R2 all had support for different buckets and individual access controls.
